README.md

```diff
@@ -100,6 +101,13 @@ To conditionally activate plugins, `vim-plug` has a
 have built-in support for `cond = vim.g.vscode`. See
 [plugins](https://github.com/vscode-neovim/vscode-neovim/wiki/Plugins) in the wiki for tips on configuring Vim plugins.
 
+#### Neovim configuration for best compatibility
+
+Though the extension strives to be as compatible as possible with older versions of Neovim, some older versions may have
+quirks that are not present anymore. In light of this, certain configuration settings are recommended in some older
+versions for the best experience. These can be found
+[on the wiki](https://github.com/vscode-neovim/vscode-neovim/wiki/Version-Compatibility-Notes).
```

Comment:
We should show a message if Nvim is older than 0.10. There is no good reason for a vscode user to use an older version of Nvim.

Reply:
Disagree, here is just a small problem

At least not right now, because version 0.10.0 has a very serious issue #2000

Reply:
So you agree after Nvim 0.10.1 is released?

Reply:
I remain neutral. I usually use the nightly version. However, vscode-neovim currently works with 0.9. It just doesn't handle some edge cases well enough.

Reply:
@justinmk I don't necessarily disagree with you. It's definitely worth noting that both VSCode and its extensions  auto-update, but nvim does not. Unless you're greatly invested in the nvim ecosystem, you may not even know to update.

That said, I don't think it would be a bad thing for us to develop some kind of "policy" around how long we support nvim versions for (x months after release?). In this case, the plugin still works, it just has a quirk in this older version of nvim, so I'm inclined to not say it's "broken".